### ISSUE TITLE
Issue with getting a page

### ISSUE THREAD

**Issue by OmerBenGera** [Anorov/cloudflare-scrape#267]
Before creating an issue, first upgrade cfscrape with `pip install -U cfscrape` and see if you're still experiencing the problem.

## Version number

Run `pip show cfscrape` and paste the output below:

```
Name: cfscrape
Version: 2.0.7
Summary: A simple Python module to bypass Cloudflare's anti-bot page. See https://github.com/Anorov/cloudflare-scrape for more information.
Home-page: https://github.com/Anorov/cloudflare-scrape
Author: Anorov
Author-email: anorov.vorona@gmail.com
License: UNKNOWN
Location: /usr/local/lib/python3.5/dist-packages
Requires: requests
Required-by:
```

## Complete exception and traceback

(*If the problem doesn't involve an exception being raised, leave this blank*)

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/cfscrape/__init__.py", line 294, in solve_challenge
    float(result)
ValueError: could not convert string to float: b'undefined'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "mc-market.py", line 151, in <module>
    main()
  File "mc-market.py", line 35, in main
    get_request(WEBSITE_URL)
  File "mc-market.py", line 126, in get_request
    respond = scraper.get(url, headers=headers, allow_redirects=allow_redirects)
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/cfscrape/__init__.py", line 128, in request
    resp = self.solve_cf_challenge(resp, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/cfscrape/__init__.py", line 181, in solve_cf_challenge
    answer, delay = self.solve_challenge(body, domain)
  File "/usr/local/lib/python3.5/dist-packages/cfscrape/__init__.py", line 297, in solve_challenge
    "Cloudflare IUAM challenge returned unexpected answer. %s" % BUG_REPORT
ValueError: Cloudflare IUAM challenge returned unexpected answer. Cloudflare may have changed their technique, or there may be a bug in the script.

```

## URL of the Cloudflare-protected page

https://www.mc-market.org/

**Comment by pro-src** [comment]
Duplicate of #248, #256, #263
